modules/duplicating-a-connection-type.adoc

```diff
@@ -0,0 +1,39 @@
+:_module-type: PROCEDURE
+
+[id="duplicating-a-connection-type_{context}"]
+= Duplicating a connection type
+
+[role='_abstract']
+
+ifdef::upstream[]
+As an {productname-short} administrator, you can create a new connection type by duplicating an existing one, as described in this procedure, or you can create a new connection type as described in link:{odhdocshome}/managing-odh/#creating-a-connection-type_resource-mgmt[Creating a connection type].
+endif::[]
+ifndef::upstream[]
+As an {productname-short} administrator, you can create a new connection type by duplicating an existing one, as described in this procedure, or you can create a new connection type as described in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing-connection-types#creating-a-connection-type_resource-mgmt[Creating a connection type].
+endif::[]
+
+You might also want to duplicate a connection type if you want to create versions of a specific connection type.
+
+.Prerequisites
+* You have logged in to {productname-short} as a user with {productname-short} administrator privileges. 
+
+.Procedure
+. From the {productname-short} dashboard, click *Settings* -> *Connection types*.
+
+. From the list of available connection types, find the connection type that you want to duplicate. 
++
+Optionally, you can select the Options menu image:images/osd-ellipsis.png[Options menu] and then click *Preview* to see how the related connection form appears to your users.
+
+. Click the Options menu image:images/osd-ellipsis.png[Options menu], and then click *Duplicate*.
++
+The *Create connection type* form appears populated with the information from the connection type that you duplicated except the *connection type name* and the *resource name* include "copy of".
```

Comment:
The end of this sentence doesn't make sense to me:

 and the *resource name* include "copy of".